musicbrainzngs/mbxml.py

```diff
@@ -505,7 +505,9 @@ def parse_disc(disc):
     result = {}
     attribs = ["id"]
     elements = ["sectors"]
-    inner_els = {"release-list": parse_release_list}
+    inner_els = {"release-list": parse_release_list,
+                 "offset-list": parse_offset_list
```

Comment:
Your docs are now out of date
https://github.com/JonnyJD/python-musicbrainzngs/blob/offset-list/musicbrainzngs/musicbrainz.py#L969